src/py/pyodide/_package_loader.py

```diff
@@ -179,3 +181,19 @@ def get_dynlibs(archive: IO[bytes], target_dir: Path) -> list[str]:
         for path in dynlib_paths_iter
         if should_load_dynlib(path)
     ]
+
+
+def sub_resource_hash(sha_256: str) -> str:
+    """Calculates the sub resource integrity hash given a SHA-256
+
+    Parameters
+    ----------
+    sha_256
+        A hexdigest of the SHA-256 sum.
+
+    Returns
+    -------
+        The sub resource integrity hash corresponding to the sum.
+    """
```

Comment:
Could you please add a doctest to illustrate how to use this (and it would serve a test)